src/main.css

```diff
@@ -0,0 +1,10632 @@
+
+.fs1 {
+  font-size: 2.5rem;
+}
+
+.custom-icon {
+  background-color: black;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+  border-radius: 50%;
+  margin-bottom: 0.5rem;
+}
+.custom-icon .icon {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  stroke-width: 0;
+  stroke: currentColor;
+  fill: currentColor;
+  background-color: #000;
+  color: white;
+  border-radius: 20%;
+}
+
+/* 
+0. Colors
+1. Main banner
+2. Nav Bar
+3. services
+4. about
+5. team
+6. portfolio
+7. blog
+8. questions
+9. testimonials
+9. contact
+10. footer
+*/
+/* COLORS */
+/* END COLORS */
+/* MAIN BANNER */
+#main-banner {
+  width: 100vw;
+  height: 50vh;
+}
+#main-banner .carousel {
+  height: 100%;
+}
+#main-banner .carousel-inner {
+  height: 100%;
+}
+#main-banner .banner-item--1 {
+  background-image: url(../resources/home-bg-2.jpg);
+  background-position: left 50% bottom 20%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-origin: border-box;
+  background-clip: border-box;
+  height: 100%;
+}
+#main-banner .banner-item--2 {
+  background-image: url(../resources/home-bg.jpg);
+  background-position: left 60% bottom 50%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-origin: border-box;
+  background-clip: border-box;
+  height: 100%;
+}
+#main-banner .banner-content {
+  text-align: center;
+  padding-top: 10vh;
+  margin-left: 2%;
+  margin: auto;
+}
+#main-banner h1 {
+  color: white;
+  font-size: 1.9em;
+  font-weight: bold;
+}
+#main-banner p {
+  color: white;
+  margin-top: 1rem;
+}
+#main-banner .bg-buttons {
+  margin-top: 2rem;
+}
+#main-banner .bg-buttons a {
+  display: inline-block;
+  background: none;
+  color: white;
+  font-weight: bold;
+  border: 3px solid white;
+  padding: 0.5rem;
+  width: 10rem;
+}
+#main-banner .bg-buttons a:hover {
+  background: #ffffff;
+  color: #64c6ae;
+  border: 3px solid #ffffff;
+  outline: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+#main-banner .bg-buttons a:focus {
+  outline: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+#main-banner .bg-buttons a:active {
+  background-color: orange;
+  border: 3px solid orange;
+  outline: none;
+  color: white;
+}
+#main-banner .bg-buttons .bg-button-2 {
+  display: none;
+}
+@media (max-height: 568px), (max-width: 280px) {
+  #main-banner {
+    height: 60vh;
+  }
+  #main-banner .bg-buttons {
+    margin-top: 1rem;
+  }
+}
+@media (min-width: 577px) {
+  #main-banner h1 {
+    font-size: 2.2em;
+  }
+  #main-banner p {
+    font-size: 1.3rem;
+  }
+  #main-banner .banner-content {
+    max-width: 500px;
+  }
+}
+@media (min-width: 577px) and (min-height: 700px) {
+  #main-banner .banner-content {
+    padding-top: 17vh;
+  }
+}
+@media (min-width: 577px) {
+  #main-banner .bg-buttons {
+    margin-top: 7%;
+  }
+  #main-banner .bg-buttons .bg-button-2 {
+    display: inline-block !important;
```

Comment:
never use important to override your own styles.
important can be used only if you are overriding other important or inline styles